doc/tutorial/download_miplib_instance.ipynb

```diff
@@ -0,0 +1,150 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MIPLIBインスタンスをダウンロードする\n",
+    "\n",
+    "OMMXリポジトリでは、MIPLIB 2017の混合整数計画問題ベンチマークインスタンスをOMMX Artifact形式のデータとして提供しています。\n",
+    "\n",
+    "```{note}\n",
+    "より詳細な説明：MIPLIB 2017のインスタンスに対応するOMMX ArtifactはOMMXリポジトリのGitHub コンテナーレジストリ [(link)](https://github.com/Jij-Inc/ommx/pkgs/container/ommx%2Fmiplib2017)で管理されています。\n",
+    "\n",
+    "GitHub コンテナーレジストリについては https://docs.github.com/ja/packages/working-with-a-github-packages-registry/working-with-the-container-registry を参照してください。\n",
+    "```\n",
+    "\n",
+    "これらのインスタンスはOMMX SDKで簡単にダウンロードでき、OMMX Adapterの入力としてそのまま利用できます。\n",
+    "例えば、MIPLIB 2017のair05インスタンス ([参照](https://miplib.zib.de/instance_details_air05.html)) をPySCIPOptで解くには、以下の2ステップで実行できます：\n",
+    "\n",
+    "1. OMMX Python SDKの`dataset`サブモジュールにある`miplib2017`関数で、air05インスタンスをダウンロードする。\n",
+    "2. ダウンロードしたインスタンスをOMMX PySCIPOpt Adapterで解く。\n",
```

Comment:
```suggestion
    "2. ダウンロードしたインスタンスをOMMX PySCIPOpt Adapterを介してPySCIPOptで解く。\n",
```

Reply:
fix in https://github.com/Jij-Inc/ommx/pull/169/commits/56efa485c7d9d0d303a8b5eea6739eaea3027bbf